scenario/LiquidationBotScenario.ts

```diff
@@ -0,0 +1,120 @@
+import { scenario } from './context/CometContext';
+import { expect } from 'chai';
+import { timeUntilUnderwater } from './LiquidationScenario';
+import { isValidAssetIndex, MAX_ASSETS } from './utils';
+// import { Liquidator } from '../build/types';
+
+const daiPool = {
+  tokenAddress: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+  poolFee: 100
+};
+
+// const LIQUIDATOR_ADDRESS = '0xb21b06D71c75973babdE35b49fFDAc3F82Ad3775'; // latest
+// const LIQUIDATOR_ADDRESS = '0x42480C37B249e33aABaf4c22B20235656bd38068'; // previous
+
+// XXX delete
+const UNISWAP_V3_FACTORY_ADDRESS = '0x1F98431c8aD98523631AE4a59f267346ea31F984';
+const WETH9 = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2';
+const RECIPIENT = '0xe8F0c9059b8Db5B863d48dB8e8C1A09f97D3B991';
+const UNISWAP_ROUTER = '0xe592427a0aece92de3edee1f18e0157c05861564';
+const SUSHISWAP_ROUTER = '0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F';
+
+enum Exchange {
+  Uniswap,
+  SushiSwap
+}
+
+for (let i = 0; i < MAX_ASSETS; i++) {
+  scenario.only(
+    `LiquidationBot > liquidates an underwater position for $asset${i}`,
+    {
+      filter: async (ctx) => ctx.world.base.network === 'mainnet' && await isValidAssetIndex(ctx, i),
+      tokenBalances: {
+        $comet: { $base: 1000 },
+      },
+      cometBalances: {
+        albert: {
+          $base: -10000,
+          [`$asset${i}`]: 20
+        },
+        betty: { $base: 1000 },
+      },
+    },
+    async ({ comet, actors }, _context, world) => {
+      const { albert, betty } = actors;
+
+      // const liquidator = await world.deploymentManager.existing(
+      //   'liquidator',
+      //   LIQUIDATOR_ADDRESS
+      // ) as Liquidator;
+
+      const liquidator = await world.deploymentManager.deploy(
```

Comment:
Deploying in a scenario is obviously not ideal.

I think we avoided adding the Liquidator contract to `roots.json` when we first developed it since we felt like it was not actually a part of Comet.

I'm thinking we should revisit that idea, though. It seems like having a functioning Liquidation contract is more critical than we'd originally thought